### ISSUE TITLE
Request.

### ISSUE THREAD

**Issue by jmdearras** [ntoff/OctoPrint-FanSpeedSlider#7]
Could you produce a plug-in to control the case light in Marlin?  M355 S1 P255 to turn on full, etc, just like the fan.  Should be identical to this one, except for verbiage and the actual command.  The limits options would not be needed.

Thanks,
Jim

**Comment by ntoff** [comment]
That can easily be done with a custom control, and without all the fancy limiting even the fan slider could have just be a custom control. The only thing that warranted it being a plugin was the ability to limit the fan's power output on the fly, without having to re-slice.

http://docs.octoprint.org/en/master/features/custom_controls.html see the "fan" section.

I've tested this and it "seems to work", it sends the command but as none of my printers run Marlin, I couldn't tell you whether or not it actually turns the lights on and off:

    controls:
      - name: Case Lights
        layout: horizontal
        children:
        - name: Adjust Lights
          command: M355 S1 P%(intensity)s
          input:
          - name: Intensity
            parameter: intensity
            default: 255
            slider:
              min: 0
              max: 255
        - name: Lights Off
          command: M355 S1 P0

Closing this as it's not really an issue with this plugin. If you want to request plugins and features, a more appropriate place would be OctoPrint's own git issue tracker, using the title prefix `[Request]` where multiple plugin authors might see it and be willing to contribute.
